src/testFixtures/patient.ts

```diff
@@ -0,0 +1,210 @@
+export const patient = {
```

Comment:
Are we actually using any of the response data in our code? If we are only use the status, I don't think it's necessary to include full response fixtures like this.